extensions/binding/org.eclipse.smarthome.binding.astro.test/src/test/groovy/org/eclipse/smarthome/binding/astro/handler/test/AstroScheduledJobsTest.groovy

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.astro.handler.test
+
+import static org.hamcrest.CoreMatchers.*
+import static org.junit.Assert.*
+
+import org.eclipse.smarthome.config.core.Configuration
+import org.eclipse.smarthome.core.thing.ThingStatus
+import org.junit.Test
+import org.eclipse.smarthome.binding.astro.AstroBindingConstants
+import org.eclipse.smarthome.binding.astro.test.AstroOSGiTest
+import org.eclipse.smarthome.binding.astro.test.AstroOSGiTest.AcceptedItemType
+import org.quartz.JobKey
+import org.quartz.Scheduler
+import org.quartz.SchedulerException
+import org.quartz.Trigger
+import org.quartz.impl.StdSchedulerFactory
+import org.quartz.impl.matchers.GroupMatcher
+import org.apache.commons.lang.StringUtils
+/**
+ * OSGi tests for the {@link AstroThingHandler}
+ * <p>
+ * This class tests the scheduling of the jobs.
+ *
+ * @author Petar Valchev - Initial Implementation
+ * @author Svilen Valkanov - Added missing configuration parameters and replaced headers
+ *
+ */
+class AstroScheduledJobsTest extends AstroOSGiTest {
```

Comment:
Considering https://github.com/eclipse/smarthome/issues/3416, maybe it's better to drop it completely.